python/ray/train/v2/tests/test_state.py

```diff
@@ -303,6 +303,8 @@ def test_train_state_manager_run_attempt_lifecycle(ray_start_regular):
     attempt = attempts["test_run"]["attempt_1"]
     assert attempt.status == RunAttemptStatus.FINISHED
     assert attempt.end_time_ns is not None
+    assert len(attempt.workers) == 2
+    assert all(w.status == ActorStatus.DEAD for w in attempt.workers)
```

Comment:
Add the same check in the errored test case: https://github.com/ray-project/ray/blob/0ca38e9a7aaeea91be0ec73618e34bfe4b5f18cc/python/ray/train/v2/tests/test_state.py#L363

Reply:
Nice nice. Updated `test_callback_worker_group_error` which tests the `RunAttempt`/`WorkerGroup` level rather than `Run`/`Controller`.